pkgs/development/compilers/go/1.20.nix

```diff
@@ -46,11 +46,11 @@ let
 in
 stdenv.mkDerivation (finalAttrs: {
   pname = "go";
-  version = "1.20.11";
+  version = "1.20.12";
```

Comment:
+1 for `go_1_20` to master.

Reply:
(Not prevented by merging this to staging-next.)